RecoEcal/EgammaClusterAlgos/src/PreshowerPhiClusterAlgo.cc

```diff
@@ -90,7 +90,6 @@ reco::PreshowerCluster PreshowerPhiClusterAlgo::makeOneCluster(ESDetId strip,
   Point pos(x_pos,y_pos,z_pos);
 
   reco::PreshowerCluster cluster = reco::PreshowerCluster(Eclust, pos, usedHits, plane);
-  used_strips = used_s;
```

Comment:
I think this simpler solution is better. The code could be re-designed, but the benefit would be minor and not worth the effort and risk of new bugs.